pyglmnet/pyglmnet.py

```diff
@@ -819,8 +875,8 @@ def predict(self, X):
             raise ValueError('Input data should be of type ndarray (got %s).'
                              % type(X))
 
-        yhat = _lmb(self.distr, self.beta0_,
-                    self.beta_, X, self.eta)
+        yhat = _lmb(self.distr, self.beta0_, self.beta_, X, self.eta,
+                    fit_intercept=True)
```

Comment:
why not `fit_intercept=self.fit_intercept`?

Reply:
see line 914

Reply:
that won't work. I suspect line 914 is not properly tested. It needs to be fixed then.